### PR TITLE
Add Trevor's dynamic config script and configurations

### PR DESCRIPTION
**Pull request by philbogle**
Check in Trevor's scripts for updating the dynamic config based on several predefined configurations.  This will allow us to easily apply the configurations appropriate to various environments.

From the README:

The `update_gatekeeper.sh` script in this directory allows you to update the
dynamic configuration with one of several predefined configurations defined in
YAML files.

For example, run the following to cache the HOC 2015 levels only and disable
milestone posts.

```
./update_gatekeeper.sh gatekeeper_hoc2015_cached.yml
```

### CODE REVIEW

**Comment by philbogle** [comment]
(Reassigning since Trevor is travelling.)

File: bin/dynamic_config/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The `update_gatekeeper.sh` script in this directory allows you to update the
```
Comment (mehalshah) [inline bin/dynamic_config/README.md:1]: Does this need to be done from a specific directory?

File: bin/dynamic_config/update_gatekeeper.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+FULL_CONFIG_PATH=`readlink -f "$1"`
+cd /home/ubuntu/adhoc/dashboard
+bundle exec ruby /home/ubuntu/dynamic_config/lib/update_gatekeeper.rb $FULL_CONFIG_PATH
```
Comment (mehalshah) [inline bin/dynamic_config/update_gatekeeper.sh]: Is this guaranteed to be the location? Can we make this a relative path so this can be run locally?

**Comment by mehalshah** [comment]
Last comment is a concern - if I'm wrong then ship it.

**Comment by trevorb** [comment]
Just got to DC.  I need to refactor this a bit so it'll work outside of an adhoc machine. I'll update this PR later tonight with changes.

File: bin/dynamic_config/README.md (hunk 1 above)
Comment (philbogle) [inline bin/dynamic_config/README.md:1]: Fixed this to work for any directory.

**Comment by philbogle** [comment]
That for catching that @mehalshah .  

@trevorb , I changed this to work from any directory, so if it looks OK no need for you to rework it.

**Comment by trevorb** [comment]
lgtm, thanks for updating it for me

**Pull request merged**